Client/Frontend/UserContent/UserScripts/CookieControl.js

```diff
@@ -0,0 +1,40 @@
+// The $<> is used to replace the var name at runtime with a random string.
+
+//Cookie should neither be saved nor accessed (local or session) when user has blocked all cookies.
+var $<cookie> = Object.getOwnPropertyDescriptor(Document.prototype, 'cookie') ||
+Object.getOwnPropertyDescriptor(HTMLDocument.prototype, 'cookie');
+if ($<cookie> && $<cookie>.configurable) {
```

Comment:
Why is `configurable` needed here but not for `localStorage` or `sessionStorage`?

Reply:
Configurable, if false, any attempts to delete the property or change its attributes  will fail. So it is just a check to make sure we don't attempt to change a non configurable property. 
LocalStorage and SessionStorage are readOnly, so not required.